main.go

```diff
@@ -21,23 +22,27 @@ import (
 var (
 	VERSION = "0.0.1"
 
-	FlagConfigFile         = "config"
-	FlagProvisionerName    = "provisioner-name"
-	EnvProvisionerName     = "PROVISIONER_NAME"
-	DefaultProvisionerName = "rancher.io/local-path"
-	FlagNamespace          = "namespace"
-	EnvNamespace           = "POD_NAMESPACE"
-	DefaultNamespace       = "local-path-storage"
-	FlagHelperImage        = "helper-image"
-	EnvHelperImage         = "HELPER_IMAGE"
-	DefaultHelperImage     = "busybox"
-	FlagServiceAccountName = "service-account-name"
-	DefaultServiceAccount  = "local-path-provisioner-service-account"
-	EnvServiceAccountName  = "SERVICE_ACCOUNT_NAME"
-	FlagKubeconfig         = "kubeconfig"
-	DefaultConfigFileKey   = "config.json"
-	DefaultConfigMapName   = "local-path-config"
-	FlagConfigMapName      = "configmap-name"
+	FlagConfigFile               = "config"
+	FlagProvisionerName          = "provisioner-name"
+	EnvProvisionerName           = "PROVISIONER_NAME"
+	DefaultProvisionerName       = "rancher.io/local-path"
+	FlagNamespace                = "namespace"
+	EnvNamespace                 = "POD_NAMESPACE"
+	DefaultNamespace             = "local-path-storage"
+	FlagHelperImage              = "helper-image"
+	EnvHelperImage               = "HELPER_IMAGE"
+	DefaultHelperImage           = "busybox"
+	FlagServiceAccountName       = "service-account-name"
+	DefaultServiceAccount        = "local-path-provisioner-service-account"
+	EnvServiceAccountName        = "SERVICE_ACCOUNT_NAME"
+	FlagKubeconfig               = "kubeconfig"
+	DefaultConfigFileKey         = "config.json"
+	DefaultConfigMapName         = "local-path-config"
+	FlagConfigMapName            = "configmap-name"
+	FlagHelperPodFile            = "helper-pod-file"
+	DefaultHelperPodFile         = "helperPod.yaml"
+	FlagNodePathType             = "node-path-type"
```

Comment:
Why do we need nodepathType?

Reply:
since @mgoltzsche needs to open mount propagation flag, i made the node-path-type flag open to enable users to decide their own node path type such as BlockDevice, Socket etc.

Reply:
i removed this flag in this pr.